content/en/docs/examples/local-testing-with-kind/index.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Testing Istio Locally With Kubernetes In Docker
+description: Example of deploying Istio locally using Kubernetes In Docker for testing and evaluation.
+weight: 10
+keywords:
+- kubernetes
+- kind
+- metallb
+- cert-manager
+- ingress
+---
+
+## About
+
+This example uses Kubernetes In Docker ([KIND][kind]) to deploy Istio locally. The cluster produced is intended to be used for further testing, evaluation and experimentation with Istio.
+
+## Deploying Kubernetes
+
+{{< tip >}}
```

Comment:
One more suggestion would be to not make this a tip, but rather move this paragraph to a `Before you begin` section and change the first sentence to: The following instructions deploy a `kind` cluster similar to ...

Reply:
Sounds good to me, changed :+1: